cores/cv32e40p/user_manual/source/control_status_registers.rst

```diff
@@ -9,135 +9,149 @@ that were needed for the PULP system. The reason for this is that we
 wanted to keep the footprint of the core as low as possible and avoid
 any overhead that we do not explicitly need.
 
-Writes of a non-supported value to a CSR does not throw an illegal
+CSR Map
+-------
+
+Table 7 below lists all
+implemented CSRs.  To columns in Table 7 may require additional explanation:
+
+The **Parameter** column identifies those CSRs that are dependent on the value
+of specific compile/synthesis parameters. If these parameters are not set as
+indicated in the Table then the associated CSR is not implemented.  If the
+parameter column is empty then the associated CSR is always implemented.
+
+The **Privilege** column indicates the access mode of a CSR.  The first letter
+indicates the lowest privilege level required to access the CSR.  Attempts to
+access a CSR with a higher privilege level than the core is currently running
+in will throw an illegal instruction exception.  This is largely a moot point
+for the CV32E40P as it only supports machine and debug modes.  The remaining
+letters indicate the read and/or write behavior of the CSR when accessed by
+the indicated or higher privilge level:
+
+* **RW**: CSR is **read-write**.  That is, csr instructions may write any value
+  and that value will be returned on a subsequent read (unless a side-effect
+  causes the core to change the CSR value).
+
+* **RO**: CSR is **read-only**.  Writes by csr instrucions have no effect.
```

Comment:
Replace text by: Writes by CSR instructions raise an illegal instructions exception.

Reply:
You are right that this is what the priviliged-spec calls for (Section 2.1).  However, I do not see this in simulation.

Reply:
Then please raise it as an RTL bug indeed